### ISSUE TITLE
Does not work against OpenSSH 7.4p1

### ISSUE THREAD

**Issue by smarek** [Knio/sshsec#1]
Hi, thanks for the tool

However it seems, that running scan against server with OpenSSH 7.4p1 (OpenSSH_7.4p1 Debian-10, OpenSSL 1.0.2h  3 May 2016) timeouts

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/sshsec/scanner.py", line 75, in scan
    ident = s.read_line().decode('ascii').strip()
  File "/usr/local/lib/python3.5/site-packages/sshsec/scanner.py", line 60, in read_line
    s = self.read(1)
  File "/usr/local/lib/python3.5/site-packages/sshsec/scanner.py", line 43, in read
    s = self.socket.recv(n)
socket.timeout: timed out
```

Error from remote server auth.log shows `sshd[25957]: Did not receive identification string from {IP} port {PORT}`

Could you please inspect the situation?

**Comment by Knio** [comment]
Hi, can you point me to a server where I could test this on?

**Comment by smarek** [comment]
Sure, my server at `46.249.37.143`

Script I'm using like `sshscan.py 46.249.37.143`

```python
#!/usr/bin/env python3

import sys
import json
from sshsec.scanner import scan

print("Scanning: %s" % sys.argv[1])

r = scan((sys.argv[1], 22))

for key,val in r["gex"].items():
    if "p" in val:
        r["gex"]["%s" % key]["p-hex"] = hex(val["p"]).upper()

print(json.dumps(r, indent=2, sort_keys=True))
```

**Comment by Knio** [comment]
This seems to fix it for me, let me know if it works

**Comment by smarek** [comment]
Yes, thank you, I've just verified the fix is working with version 1.3.2
`pip3 install sshsec --upgrade --user` to upgrade correctly picked up latest version :+1:
